config/application.rb

```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
 
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{yml}')]
+
+    routes.default_url_options[:host] = Figaro.env.mailer_domain_name
```

Comment:
It works for me. We'll need to update `./spec/jobs/sms_sender_number_change_job_spec.rb` to replace `'http://localhost:3000/contact'` with `'http://www.example.com/contact'` since that's what `domain_name` is set to in the test environment in `application.yml`